web/src/App.jsx

```
@@ -3143,61 +3342,210 @@ function App() {
             </>
           )}
+                  </>)}
+                </>
+              )}
         </div>
       </div>
+
+      {/* Contact Info Modal */}
+      {viewingContactAddress && (
+        <div className="contact-modal-overlay" onClick={() => setViewingContactAddress(null)}>
+          <div className="contact-modal" onClick={(e) => e.stopPropagation()}>
```

Biome (2.1.2)

[error] Static Elements should not be interactive.

To add interactivity such as a mouse or key event listener to a static element, give the element an appropriate role value.

(lint/a11y/noStaticElementInteractions)